crowdin.yml

```diff
@@ -9,6 +9,15 @@ files:
       - /docs/src/pages/discover-more/roadmap/*
       - /docs/src/pages/company/**/*
     translation: /%original_path%/%file_name%-%two_letters_code%.%file_extension%
+  - source: /docs/data/**/*.md
+    ignore:
+      - /**/%file_name%-%two_letters_code%.md
+      - /docs/data/material/getting-started/templates/*/**/*
+      - /docs/data/material/discover-more/backers/*
+      - /docs/data/material/discover-more/roadmap/*
+      - /docs/data/material/premium-themes/*/**/*
+      - /docs/data/company/**/*
```

Comment:
This stays top level, outside of the docs.

```suggestion
```

Reply:
> outside of the docs

Sorry, I don't get it. Can you explain a bit more? Why this line is not needed?

Reply:
The company pages are not docs pages. Their location should not change, they are, and wills stay in `/docs/src/pages/company/`

Reply:
Alright.

Reply:
But if the purpose of `/data/` is to host the "data" regardless of the page, then yes, agree, it makes sense to move the open roles, and contact content to it.

Reply:
Yes, that is my main goal to move markdown and demos to data folder